Dynamics/createMassAndCoriolisMatrixFunction.m

```diff
@@ -0,0 +1,47 @@
+function [HFunction,HDotFunction,CFunction]= createMassAndCoriolisMatrixFunction(robotURDFModel,geneate_c_code,location_generated_fucntion)
```

Comment:
```suggestion
function [HFunction,HDotFunction,CFunction]= createMassAndCoriolisMatrixFunction(robotURDFModel,generate_c_code,location_generated_fucntion)
```

Reply:
Done in https://github.com/robotology/urdf2casadi-matlab/pull/6/commits/33e8b94b0b5c31f272bd67201a31e738e16e4ca1